test/integration/overload_integration_test.cc

```diff
@@ -936,28 +953,38 @@ TEST_P(LoadShedPointIntegrationTest, HttpDownstreamFilterLoadShed) {
   EXPECT_EQ(response_that_is_proxied->headers().getStatusValue(), "200");
 }
 
-TEST_P(LoadShedPointIntegrationTest, ListenerAcceptDoesNotShedLoadWhenBypassed) {
+TEST_P(LoadShedPointIntegrationTest, HttpConnManagerDoesNotShedLoadWhenBypassed) {
```

Comment:
I think we still want a test that works on a tcp_listener e.g. to validate that those listeners don't get disabled or connections rejected. 

consider a test using `envoy.overload_actions.reject_incoming_connections` where listener 1 bypasses and listener 2 does not.
1. trigger overload state
2. send request to listener 1 validate it goes through
3. send request to listener 2 validate the connection is rejected

Reply:
Added the test